resources/jscomposition/cases/casesDetail/config/columns.js

```diff
@@ -66,6 +70,7 @@ const requestNumberColumn = () => ({
   field: "id",
   header: "Request #",
```

Comment:
Please update to `Request ID`
![image](https://github.com/user-attachments/assets/1664711a-887a-4617-8e7d-f16c78d3a73b)